src/test/java/com/gooddata/JsonMatchers.java

```diff
@@ -37,6 +41,16 @@ private JsonMatchers() {
         return new JsonFileMatcher<>(expectedResourceName);
     }
 
+    /**
+     * Creates a matcher that matches String against JSON in given resource file.
+     * @param expectedResourceName name of resource (relative patch in resource folder)
+     * @return  matcher that matches String against JSON in given resource file
+     */
+    @Factory
+    public static Matcher<String> isJsonString(final String expectedResourceName) {
```

Comment:
well, to me it seems perfectly correct since it matches String against the String read from given resource. Can you please explain what else could the user of this matcher expect? Or just recommend better name...